From: Sun YangKai <sunk67188@gmail.com>
Subject: mm: zswap: use ATOMIC_LONG_INIT to initialize zswap_stored_pages
Date: Wed, 26 Feb 2025 16:04:09 +0800

This is currently the only atomic_long_t variable initialized by
ATOMIC_INIT macro found in the kernel by using

	grep -r atomic_long_t | grep ATOMIC_INIT

Link: https://lkml.kernel.org/r/20250226080410.10433-1-sunk67188@gmail.com
Signed-off-by: Sun YangKai <sunk67188@gmail.com>
Cc: Chengming Zhou <chengming.zhou@linux.dev>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Nhat Pham <nphamcs@gmail.com>
Cc: Yosry Ahmed <yosry.ahmed@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/zswap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/zswap.c~mm-zswap-use-atomic_long_init-to-initialize-zswap_stored_pages
+++ a/mm/zswap.c
@@ -43,7 +43,7 @@
 * statistics
 **********************************/
 /* The number of compressed pages currently stored in zswap */
-atomic_long_t zswap_stored_pages = ATOMIC_INIT(0);
+atomic_long_t zswap_stored_pages = ATOMIC_LONG_INIT(0);
 
 /*
  * The statistics below are not protected from concurrent access for
_
